aperturedb/cli/utilities.py

```diff
@@ -24,10 +25,15 @@ def confirm(command: CommandTypes):
     return True
 
 
+def print_utils(utils):
+    print(utils)
+
+
 @app.command()
 def execute(command: CommandTypes):
     utils = Utils(create_connector())
     available_commands = {
+        CommandTypes.STATUS: lambda: print_utils(utils),
```

Comment:
```suggestion
        CommandTypes.STATUS: lambda: print(utils),
```

Reply:
oh, nice. I tried exactly this first, but I was getting and error. That's why I had to add that extra function (I didn't like it either). Would you try it and tell me if it works for you?